sh.exe.stackdump

```diff
@@ -0,0 +1,22 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FF8D7F2AAB7
```

Comment:
You are right, it is not. How can I remove it?

It seems that checks failed, however according to console output my tests passed. What should I do in this case?